src/main/resources/db/migration/V1.19.0__Create_break_trigger_preference_table.sql

```diff
@@ -0,0 +1,9 @@
+USE seichiassist;
+
+CREATE TABLE player_break_trigger_preference(
+    uuid CHAR(36) NOT NULL,
+    trigger_category ENUM('ManaFullyConsumed') NOT NULL,
+    do_not_break BOOL NOT NULL DEFAULT FALSE,
+    PRIMARY KEY(uuid, trigger_category),
+    INDEX index_player_break_trigger_preference_on_uuid (uuid)
+);
```

Comment:
このテーブルで扱うデータは「マナがないときにスキルの発動を抑制するかどうか」ということだけなので、`trigger_category` カラムが `ENUM` で複数の値を持つことができる必要がないと思います(あくまで設定が有効か無効かの2値のみを扱うので)
なので、以下のようなテーブルにすると良いと思います。

```suggestion
CREATE TABLE player_break_suppression_preference(
    uuid CHAR(36) NOT NULL,
    do_break_suppression_due_to_mana BOOL NOT NULL DEFAULT FALSE,
    FOREIGN KEY fk_player_break_suppression_preference_uuid REFERENCES playerdata(uuid)
);
```